app/src/main/java/in/avimarine/boatangels/db/iDb.java

```diff
@@ -23,6 +25,8 @@
 
   void addInspection(Inspection inspection);
 
+  void addusers(User user);
```

Comment:
This should be called addUser (camelCase) and not users since you are only adding one user here.

Reply:
done